projects/mysadna/src/app/app.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="app-header">
+  <div class="app-logo">
+    <img src="../assets/images/pkw-logo.png" class="my-sadna-logo">
```

Comment:
s / `../assets/images/pkw-logo.png` / `/assets/images/pkw-logo.png`
no need to use `..`

Reply:
Fixed.